File: platforms/web/examples/geometry/mesh/clear_indices.js
import { Mesh, Vertex } from "fragmentcolor";

const mesh = new Mesh();
mesh.addVertices([ new Vertex([-0.5, -0.5]), new Vertex([ 0.5, -0.5]), new Vertex([ 0.0,  0.5]), ]);
mesh.setIndices([0, 1, 2]);
mesh.clearIndices(); // back to auto-derived dedup;

File: platforms/python/examples/geometry/mesh/clear_indices.py
from fragmentcolor import Mesh, Vertex

mesh = Mesh()
mesh.add_vertices([
    Vertex([-0.5, -0.5]),
    Vertex([ 0.5, -0.5]),
    Vertex([ 0.0,  0.5]),
])
mesh.set_indices([0, 1, 2])
mesh.clear_indices(); # back to auto-derived dedup